Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -1636,19 +1694,14 @@ void qMRMLSliceControllerWidget::setSliceOrientation(const QString& orientation)
 {
   Q_D(qMRMLSliceControllerWidget);
 
-#ifndef QT_NO_DEBUG
-  QStringList expectedOrientation;
-  expectedOrientation << "Axial" << "Sagittal" << "Coronal" << "Reformat";
-  Q_ASSERT(expectedOrientation.contains(orientation));
-#endif
-
   if (!d->MRMLSliceNode || !d->MRMLSliceCompositeNode)
     {
     return;
     }
 
   d->SliceLogic->StartSliceNodeInteraction(vtkMRMLSliceNode::OrientationFlag);
   d->MRMLSliceNode->SetOrientation(orientation.toLatin1());
+  this->fitSliceToBackground();
```

Comment:
It is not required. However I thought it can be useful. For example:
1) if the datacube has 3 dimensions very different (so not cubic)
2) the user change orientation
3) the view can be that it is not "fitting" the whole slice
4) the user has to click on the widget to fit the view to the slice

line 1704 make automatic  point 4. However, now I am thinking this can be annoying if the user has zommed on a portion of data and does not want to "refit" after chaning the orientation.

Up to you if to leave it or remove it. @lassoan ?